src/Symfony/Component/Translation/Command/TranslationPullCommand.php

```diff
@@ -95,6 +95,7 @@ protected function configure(): void
                 new InputOption('domains', null, InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY, 'Specify the domains to pull.'),
                 new InputOption('locales', null, InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY, 'Specify the locales to pull.'),
                 new InputOption('format', null, InputOption::VALUE_OPTIONAL, 'Override the default output format.', 'xlf12'),
+                new InputOption('as-tree', null, InputOption::VALUE_OPTIONAL, 'Associated to --format=yaml, it will write messages as a tree-like structure. The given value defines the level where to switch to inline YAML'),
```

Comment:
> The given value defines the level where to switch to inline YAML

I dont see this in the PR

Reply:
I forgot to add the 'inline' option of YamlFileDumper. It is now passed too. Thank you for your feedbacks